src/data/markdown/translated-guides/en/01 Getting started/04 Results Output.md

```diff
@@ -3,150 +3,92 @@ title: 'Results output'
-```
-
-</CodeGroup>
-
-> To learn more about the metrics k6 collects and reports, read the [Metrics guide](/using-k6/metrics).
-
-### Trend metrics
+The end-of-test-summary shows aggregated statistical values for your result metrics, including:
+- Median and average values
+- Minimum and maximum values
+- p90, p95, and p99 values
 
-[Trend metrics](/using-k6/metrics#metric-types) collect trend statistics (min/max/avg/percentiles) for a series of values.
-On stdout, they are printed like this:
+If this default report is unsuitable, you can change the trend stats with
+the [`--summary-trend-stats`](https://k6.io/docs/using-k6/k6-options/reference#summary-trend-stats) option.
+You can also change the time unit with 
+the [`--summary-time-unit`](/using-k6/k6-options/reference#summary-time-unit) option.
 
-<CodeGroup labels={[]}>
+For example, this command displays only the median, p95, and p99.9 values,
+and it presents the statistics in milliseconds instead of seconds.
 
-```bash
-http_req_duration..........: avg=143.14ms min=112.87ms med=136.03ms max=1.18s    p(90)=164.2ms  p(95)=177.75ms
+```sh
+k6 run --iterations=100 --vus=10 \
+--summary-trend-stats="med,p(95),p(99.9)" --summary-time-unit="ms" script.js
 ```
 
-</CodeGroup>
-
-To change the stats reported for `Trend` metrics, you can use the [`summaryTrendStats` option](/using-k6/options#summary-trend-stats).
-You can also make k6 display time values with a fixed time unit (seconds, milliseconds or microseconds) via the [`summaryTimeUnit` option](/using-k6/options#summary-time-unit).
-And, as mentioned above, you can completely customize the whole report via the [`handleSummary()` callback](/results-visualization/end-of-test-summary#handlesummary-callback).
+<Blockquote mod="note" title="">
 
-<CodeGroup labels={[]}>
+With a single iteration and VU, the min, max, median, average, and p values would all be the same.
 
-```bash
-$ k6 run --summary-trend-stats="min,avg,med,p(99),p(99.9),max,count" --summary-time-unit=ms  script.js
-```
+</Blockquote>
 
-</CodeGroup>
 
-### Export summary
+### Custom reports with `handleSummary()`
 
-To get your aggregated test results in a machine-readable format,
-you can export the end-of-test summary report to a JSON file.
-This helps with things like integration with dashboards, external alerts, etc.
+At the end of the test, k6 automatically creates an object with all aggregated statistics.
+To completely customize the end-of-test summary,
+you can use the `handleSummary()` function to process this object into any text format.
 
-We recommend the [`handleSummary()` callback](/results-visualization/end-of-test-summary#handlesummary-callback).
-It lets you customize the end-of-test summary and export the report data to any desired format (e.g. JSON, CSV, XML (JUnit/xUnit/etc.), HTML, TXT, etc.).
+For example, the community project [k6 reporter](https://github.com/benc-uk/k6-reporter) uses `handleSummary()` to make an HTML report from your k6 summary metrics.
```

Comment:
We can remove this sentence and add the link previously when listing the HTML example.

Reply:
It's one paragraph now.